src/solver/optimisation/opt_construction_variables_optimisees_lineaire.cpp

```diff
@@ -88,6 +88,29 @@ void OPT_ConstruireLaListeDesVariablesOptimiseesDuProblemeLineaire(PROBLEME_HEBD
                 NombreDeVariables++;
             }
 
+            for (const auto& storage : (*problemeHebdo->ShortTermStorage)[pays].storages)
```

Comment:
As you see, function **OPT_InitialiserLesBornesDesVariablesDuProblemeLineaire(...)** and the source file it belongs to are growing.
A possible quite simple solution for them to stop growing (to be discussed) would be to have : 
```C++
void OPT_InitialiserLesBornesDesVariablesDuProblemeLineaire(PROBLEME_HEBDO* problemeHebdo, ...)
{
      problemeHebdo->hydroModeller.setBounds();
      problemeHebdo->thermalModeller.setBounds();
      ...
      // And of course
      problemeHebdo->shortTermStorageModeller.setBounds();
      ...
}
```
Several remarks about this :
- This solution should be discussed to determine if it is suitable or feasible 
- It intends to extract bound management in a more specialized class.
- For this evolution, only the `problemeHebdo->shortTermStorageModeller.setBounds();` would be taken care of
- This solution would be applicable to any other functions in the **OPT** part
- It would be applicable to any other production asset taking a part in the optimization 
- Class **ShortTermstorageModeller** would have a constructor taking as argument  a PROBLEME_HEBDO and characteristics on short term storage.

Reply:
I totally agree with this remark, can you open a refactor issue on this ?

Reply:
Yes I can. I will.
Honestly, what I have in mind, eventually, is something like (you probably guessed already) : 
 ```C++
 // Only for setting bounds
 for (auto& asset : list_of_assets)
     asset->setBounds();
```